frontend/elements/src/entry/home/sections/footer-productservices.ts

```diff
@@ -0,0 +1,66 @@
+import { MEDIA_UI } from '@utils/path';
+import { LitElement, html, css, customElement, property } from 'lit-element';
+import "@elements/entry/home/TOSORT/column-list";
+@customElement('footer-productservices')
+export class _ extends LitElement {
+  static get styles() {
+    return [css`
+    h3{
+      font-family: Poppins;
+      font-size: 20px;
+      font-weight: 800;   
+      color:#ffffff;
+    
+    }
+   main{
+     
+   }
+   
+   ul{
+    list-style-type: none;
+    margin:0;
+    padding:0;
+   }
+
+
+  .list{
+    display:block;
+    margin-top:8px;
+ }
+  
+
+    `];
+  }
+
+
+  @property()
+  head_title:string = ""; 
+
+
+ 
+
+  render() {
+
+    const {head_title} = this;
+
+    return html`
+    <main>
+    <h3>Products & Services</h3>
+    <ul>
+    <column-list text_line="Teachers" color="white" class="list" ></column-list>
```

Comment:
should be separated into STR_ and outside of the render function (though note that this one will probably get folded into a general issue of creating homepage product cards)